move-to-sdk/app/io/sphere/sdk/facets/FlexibleSelectFacetBuilder.java

```diff
@@ -0,0 +1,84 @@
+package io.sphere.sdk.facets;
+
+import io.sphere.sdk.models.Builder;
+import io.sphere.sdk.search.TermFacetResult;
+import io.sphere.sdk.search.TermModel;
+import io.sphere.sdk.search.UntypedSearchModel;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+public final class FlexibleSelectFacetBuilder<T> extends BaseSelectFacetBuilder<FlexibleSelectFacetBuilder<T>> implements Builder<FlexibleSelectFacet<T>> {
```

Comment:
It's just a builder, I didn't think builders needed Javadoc... JVM SDK's builders don't have any.